docs/LorisPHPMD.xml

```diff
@@ -16,4 +16,9 @@
 			about them not being used -->
 		<exclude name="UnusedFormalParameter" />
 	</rule>
+
+    <rule ref="rulesets/design.xml">
```

Comment:
This change doesn't seem to be doing the same thing as the PR description and code changes say. The unused variables rules are in a different ruleset from what I can tell, not the design one.

Reply:
> **Adds the modules directory to PHPMD** to mark unused variables as well as
leftover debug statements from developers...

The Unused variable rules are already active, but the `modules/` directory was excluded from analysis.

> **Also adds the PHPMD "Design Rules"** with the exception of ExitExpression and
CouplingBetweenObjects...